core/source_map.rs

```diff
@@ -33,12 +34,14 @@ pub type SourceMapData = Cow<'static, [u8]>;
 pub struct SourceMapper {
   // TODO(bartlomieju): I feel like these two should be cleared when Isolate
   // reaches "near heap limit" to free up some space. This needs to be confirmed though.
-  maps: HashMap<String, Option<SourceMap>>,
+  maps: HashMap<String, Option<Arc<SourceMap>>>,
```

Comment:
Ouch, now I'm a bit confused - what's the difference between `maps` and `native_source_maps`? Since they look almost identical is there a reason to keep them separate?

Reply:
Oh! Actually, we don't need `native_source_maps`. I remove it and we're using maps directly.